src/ophys_etl/transforms/convert_rois.py

```diff
@@ -152,12 +160,18 @@ def binarize_and_create(self):
 
         binarized_coo_rois = []
         for coo_roi in coo_rois:
-            binary_mask = binarize_roi_mask(coo_roi,
-                                            self.args['abs_threshold'],
-                                            self.args['binary_quantile'])
-            binarized_coo_rois.append(binary_mask)
+            # check if s2p border artifact
```

Comment:
I suggest the linear filtering be a discrete function. Will make testing much simpler.

Maybe something similar to:
```
def filter_rois_by_linear_dimension(rois: List[coo_matrix], threshold: int) -> List[coo_matrix]:
```

Reply:
I also wonder if there could be a more descriptive name than `linear dimension filter`.

Maybe `filter_rois_by_longest_edge_length`?

Reply:
I moved the filter function to it's own new module and renamed it to your suggested name!